website/docs/reference/transforms/dedupe.md.erb

```diff
@@ -0,0 +1,35 @@
+<%- component = metadata.transforms.dedupe -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+<%- if component.requirements.any? -%>
+## Requirements
+
+<%= component_requirements(component) %>
+
+<%- end -%>
+## Options
+
+<%= fields(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list, heading_depth: 3) %>
+
+<%- end -%>
+<%- if component.output.log -%>
+## Output
+
+<%= component_output(component, component.output.log, heading_depth: 3) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
```

Comment:
Can we include a section here explaining that it's an LRU cache and how an LRU cache works in basic terms?

Reply:
I’d also like a section on memory usage and how users can roughly estimate memory usage based on their settings.